src/systems/logical_audio_sensor_plugin/LogicalAudioSensorPlugin.hh

```diff
@@ -0,0 +1,149 @@
+  ///       See logical_audio::AttenuationShape for a list of valid
+  ///       attenuation shapes, and logical_audio::SetAttenuationShape for how
+  ///       to specify an attenuation shape in SDF.
+  ///     * <inner_radius> The inner radius of the attenuation shape.
+  ///       This value must be >= 0.0. The volume of the source will be
+  ///       <audiosource><volume> at locations that have a distance <= inner
+  ///       radius from the source.
+  ///     * <falloff_distance> The falloff distance. This value must be greater
+  ///       than the value of the source's <inner_radius>. This defines the
+  ///       distance from the audio source where the volume becomes 0.
+  ///     * <volume_level> The volume level emitted from the source. This must
+  ///       be a value between 0.0 and 1.0 (representing 0% to 100%).
+  ///     * <playing> Whether the source should play immediately or not.
+  ///       Use true to initiate audio immediately, and false otherwise.
+  ///     * <play_duration> The duration (in seconds) audio is played from the
+  ///       source. This value must be >= 0. A value of 0 means that the source
+  ///       will play for an infinite amount of time.
+  ///
+  /// Specifying a microphone via SDF is done as follows:
+  ///
+  /// <microphone> A new microphone in the environment,
+  ///   which has the following child elements:
+  ///     * <id> The microphone ID, which must be unique and >= 0.
+  ///     * <pose> The pose, expressed as "x y z roll pitch yaw".
+  ///       Each pose coordinate must be separated by whitespace.
+  ///       The pose is defined relative to the plugin's parent element.
+  ///       So, if the plugin is used inside of a <model> tag, then the
+  ///       source's <pose> is relative to the model's pose.
+  ///       If the plugin is used inside of a <world> tag, then the source's
+  ///       <pose> is relative to the world (i.e., <pose> specifies an
+  ///       absolute pose).
+  ///     * <volume_threshold> The minimum volume level the microphone
+  ///       can detect. This must be a value between 0.0 and 1.0
+  ///       (representing 0% to 100%).
+  ///
+  /// Sources can be started and stopped via ignition service calls.
+  /// If a source is successfully created, the following services will be
+  /// created for the source (where <id> is the value specified in the source's
+  /// <id> tag from the SDF):
+  ///     * /play_source_<id>
+  ///         * Starts playing the source with the specified ID.
+  ///           If the source is already playing, nothing happens.
+  ///     * /stop_source_<id>
+  ///         * Stops playing the source with the specified ID.
+  ///           If the source is already stopped, nothing happens.
+  ///
+  /// Microphone detection information can be retrieved via ignition topics.
+  /// Whenever a microphone detects a source, it publishes a message to the
+  /// /mic_<id>_detection topic, where <id> is the value specified in the
```

Comment:
Can you change the detection topic to `/mic_<id>/detection`? This is a bit of future proofing. For example, we may add additional topics to microphones, such as status or info, and it would be nice to have these topics scoped to the microphone.